x/token/types/types_test.go

```diff
@@ -0,0 +1,116 @@
+package types
+
+import (
+	"os"
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(m *testing.M) {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount("panacea", "panaceapub")
+	config.Seal()
+
+	os.Exit(m.Run())
+}
```

Comment:
@youngjoon-lee 
What kind of test is this?

Reply:
https://golang.org/pkg/testing/#hdr-Main
This is something like `setUp()` in JUnit.
The `m.Run()` executes all tests in that file. Before that, we can run some initialization logics.

In this file, I called `config.SetBech32PrefixForAccount("panacea", "panaceapub")` because it's needed for [sdk.AccAddressFromBech32](https://github.com/medibloc/panacea-core/pull/124/files#diff-b3d3e6a05fd122fd9200d46ef1f333b675e0b16cfa756dbb2f979f21b5d135ffR29).
Actually, the `SetBech32PrefixForAccount` is already being called in the `app/prefix.go`, but it cannot be imported to `x/**/*_test.go` (due to the circular import).

In near future, we need to have a common test initialization function.

Reply:
OK. Thanks!